### PR TITLE
[WIP]:Added a Blog page

### PR DESCRIPTION
**Pull request by Arioum**
## Related Issue

[Work in Progress] Adding a Blogs page with blog card items

Closes: #169

## Description of Changes

- Added a new route in pages folder `blogs.page.js`
- Added a new folder under `components` called BlogCard which contains two new files - `BlogCard.js` , `blogCard.module.css`

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| Unavailable | image |

### CODE REVIEW

**Comment by Arioum** [comment]
This is a work in progress PR. It was raised early as per request of the PA.

File: pages/blogs.page.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    title: "How to be a self-taught Full Stack Engineer",
+    content: "Lorem ipsum dolor sit amet consectetur adipisicing elit. Ab eaque odio porro dolorum? Beatae quibusdam iure quam, tempora ipsam, facilis reprehenderit aliquam qui nemo facere nam officia esse perferendis ratione.",
+    likes: "264",
+    comments: "22",
+    blogImg: "https://images.unsplash.com/photo-1618335829737-2228915674e0?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1740&q=80",
+    authorName: "Anthony Simons",
+    authorImg: "https://images.unsplash.com/photo-1552374196-c4e7ffc6e126?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2487&q=80",
+    readTime: "4"
+  },
+  {
```
Comment (anmode) [inline pages/blogs.page.js:21]: please, create a separate JSON file for now.

**Review by anmode — COMMENTED** [review]
looks cool! I'll think if the current structure works or not...but please keep in mind we would we write a blog section too! so data will be stored in the database. API part we will see. But you continue frontend keep in mind that. Thanks

**Comment by anmode** [comment]
@Arioum it would be great if you continue using tailwind css

**Comment by AmanJain18** [comment]
@anmode Check dm in discord

File: pages/blogs.page.js (hunk 1 above)
Comment (Arioum) [reply pages/blogs.page.js:21]: alright!

**Comment by Arioum** [comment]
Hey! just a heads up. I'll finish this by the end of this week. I just have exams the next 3 days!

**Comment by anmode** [comment]
@Arioum No worries!. when your exams get over. First we will talk on design part and after then you can start coding to improve the UI.

**Comment by anmode** [comment]
@Arioum ?

**Comment by Arioum** [comment]
Hello @anmode , the sem exams have begun. It will conclude on 28th. Sorry for this!

**Pull request closed without merging**